lib/mix/lib/mix/release.ex

```diff
@@ -83,7 +83,7 @@ defmodule Mix.Release do
     {include_erts, opts} = Keyword.pop(opts, :include_erts, true)
     {erts_source, erts_version} = erts_data(include_erts)
```

Comment:
Let's instead change `erts_data` to return the `erts_lib_dir` too, so we don't have to process `include_erts` twice.